scripts/log-templates/log-config-greppable.yaml

```diff
@@ -1,3 +1,3 @@
 # This template is used to grep the output.
 
-termSeverity: Debug
+termSeveritiesOut: Debug+
```

Comment:
You can write just `termSeveritiesOut: All` to not forget update this config in future if we add more severities in `log-warper`.